lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -759,13 +760,12 @@ sub _failed_check {
                    size => 15,
            }};
         push @$rows, $row;
-        $hiddens->{"id_$count"} = $row->{$check->id_column};
         ++$count;
+        $hiddens->{"id_$count"} = $row->{$check->id_column};
```

Comment:
For updates, that means updating the last one twice or trying to access an invalid id, for inserts, it would means duplicated entries.